Sources/TokamakCore/Views/Controls/Button.swift

```diff
@@ -103,7 +103,7 @@ public struct _Button<Label>: View where Label: View {
   public let action: () -> ()
 
   @State
-  public var isPressed = false
+  public var isPressed: (down: Bool, inside: Bool) = (false, false)
```

Comment:
I'm not sure about the public api here. It's a breaking change in theory, but AFAIK `_Button` should only be used internally.